src/App.vue

```diff
@@ -1,9 +1,9 @@
 <template lang="pug">
   v-app
-    Header
     v-content
-      v-container(fluid fill-height)
-        router-view
+      div(v-if="!isHome")
```

Comment:
Any reason for adding this div?

Reply:
If we are on konnekt page then the header will be hidden.

Reply:
`Header(v-if="!isHome")` won't work?

Reply:
yeah it will. didn't think of that. sorry. Will change that in the next commit.